pycsw/wsgi.py

```diff
@@ -92,6 +92,9 @@ def application(env, start_response):
                 "the server does not specify the 'gzip_compresslevel' option. "
                 "Returning an uncompressed response..."
             )
+        except configparser.NoSectionError:
```

Comment:
This `except` clause seems unnecessary here.

Since the server has been already instantiated on line [76](https://github.com/geopython/pycsw/pull/529/files#diff-27dec3b41213ad9c90fc679f954cfaf3L76) and its `__init__` method is [already catching any exceptions with trying to read the config](https://github.com/geopython/pycsw/pull/529/files#diff-e754e02805f3a01c54bfcf3bcd61c6f2R118) it seems this except clause will never be reached.